### PR TITLE
refactor: score name

### PR DESCRIPTION
**Pull request by pushkiins**
## Description:
refactor score name

### CODE REVIEW

**Review by AntonL-tech — CHANGES_REQUESTED** [review]
You have typo on Description, please fix it.

File: js/game.js [hunk 1]
```diff
@@ -78,7 +78,7 @@ function draw() {
 
  ctx.fillStyle = "#000";
  ctx.font = "24px Verdana";
- ctx.fillText("Счет: " + score, 10, cvs.height - 20);
```
Comment (AntonL-tech) [inline js/game.js:81]: Change russian words to english

**Review by AntonL-tech — CHANGES_REQUESTED** [review]

File: js/game.js (hunk 1 above)
Comment (pushkiins) [reply js/game.js:81]: done

**Review by AntonL-tech — APPROVED** [review]
@pushkiins Great job!

**Pull request merged**